builtin/mainmenu/settings/dlg_settings.lua

```diff
@@ -456,6 +456,7 @@ local function get_formspec(dialogdata)
 	local scrollbar_w = core.settings:get_bool("enable_touch") and 0.6 or 0.4
 
 	local left_pane_width = core.settings:get_bool("enable_touch") and 4.5 or 4.25
+	local left_pane_padding = 0.2
```

Comment:
This should be 0.25, as that is the smallest padding used as elsewhere

```suggestion
	local left_pane_padding = 0.25
```